### PR TITLE
presidio analyzer main fix

### PR DESCRIPTION
**Pull request by gillesdami**
see issue #15

### CODE REVIEW

**Comment by omri374** [comment]
Hi @gillesdami, thanks for your contribution. Could you please sign the CLA?

**Comment by gillesdami** [comment]
> Hi @gillesdami, thanks for your contribution. Could you please sign the CLA?

This is done, thanks you for considering my pull request.

File: presidio_evaluator/presidio_local_analyzer.py [hunk 1]
```diff
@@ -113,25 +113,24 @@ def predict(self, sample: InputSample) -> List[str]:
...
     print("Precision per entity: {}".format(entity_precision))
     print("Recall per entity: {}".format(entity_recall))
     #
-    FN_mistakes = [mistake for mistake in flatten(errors) if mistake[0:2] == "FN"]
-    FP_mistakes = [mistake for mistake in flatten(errors) if mistake[0:2] == "FP"]
+    FN_mistakes = [str(mistake) for mistake in errors if mistake.error_type == "FN"]
+    FP_mistakes = [str(mistake) for mistake in errors if mistake.error_type == "FP"]
     other_mistakes = [
-        mistake for mistake in flatten(errors) if "Wrong entity" in mistake
+        str(mistake) for mistake in errors if not mistake.error_type in ["FN", "FP"]
```
Comment (omri374) [inline presidio_evaluator/presidio_local_analyzer.py:133]:
```suggestion
        str(mistake) for mistake in errors if mistake.error_type not in ["FN", "FP"]
```

**Comment by gillesdami** [comment]
I edited my last commit to reflect the changes you suggested:
- file renamed
- class renamed
- line 133 fix

**Comment by omri374** [comment]
Thanks @gillesdami! Could you please verify that all test pass by running `pytest`?

**Comment by gillesdami** [comment]
I updated test_presidio_analyzer.py to match the new file and class name.
The test passed successfully returning "45 passed, 8 skipped in 51.93s"

**Comment by omri374** [comment]
Awesome! Thank you. Merging now.

**Pull request merged**